tests/integration/components/lt-body-test.js

```diff
@@ -24,7 +24,7 @@ test('it renders', function(assert) {
   assert.equal(this.$().text().trim(), '');
 });
 
-test('row selection', function(assert) {
+test('row selection - enable or disable', function(assert) {
```

Comment:
rather than have three tests called 'row selection' I took the liberty of adding additional detail to these test names to differentiate them